modules/exploits/linux/http/trueonline_p660hn_v2_rce.rb

```diff
@@ -98,8 +98,8 @@ def exploit
     @cookie = rand_text_alpha_lower(7)
 
     res = send_request_cgi({
+      'method' => "GET",
```

Comment:
This request was originally a POST, so you need to keep that. What `msftidy` is telling you is to use `vars_get` for the query string. The request already uses `vars_post` for the POST body.